### ISSUE TITLE
Can't send payment to lnd node

### ISSUE THREAD

**Issue by holzeis** [get10101/10101#327]
Tested on local setup with an ios simulator.

### How to reproduce:
1. start up initial setup with a funded wallet.
2. attach a shell to the lnd service and create an invoice
```
lncli --no-macaroons --network regtest addinvoice 2341
```
3. Navigate to the send payment screen on your phone and paste the payment request.
4. Hit send payment.

### Expected Result
Payment should be sent to the lnd node

### Observed Result
```
flutter: | 2023-03-23 12:57:16.977 | LogLevel.INFO | lightning::ln::onion_utils: Onion Error[from 02dd6abec97f9a748bf76ad502b004ce05d1b2d1f43a9e76bd7d85e767ffb022c9: temporary_channel_failure(0x1007)] Node indicated the outgoing channel is unable to handle the HTLC temporarily
flutter: | 2023-03-23 12:57:16.987 | LogLevel.DEBUG | lightning::routing::gossip: Updating channel with channel_update from a payment failure. Channel 139637976793088 is enabled.
flutter: | 2023-03-23 12:57:16.988 | LogLevel.DEBUG | lightning::routing::scoring: Not able to penalize channel with SCID 1099638571011 as we do not have graph info for it (likely a route-hint last-hop).
flutter: | 2023-03-23 12:57:16.988 | LogLevel.DEBUG | lightning::routing::scoring: Setting max liquidity of SCID 139637976793088, towards NodeId(024aa53abbff7055313048c20542d9656bdb0052429b9602b47310ad9a34a318d2) from 9000000000 to 2325000
flutter: | 2023-03-23 12:57:16.988 | LogLevel.INFO | Received event
flutter: | 2023-03-23 12:57:16.994 | LogLevel.INFO | Received event
flutter: | 2023-03-23 12:57:16.994 | LogLevel.WARNING | Failed to send payment to payment hash "4c43c3daf20e81c0d828ed5e0bf80fcd685392d667b2cf11854924317ed0db62": exhausted payment retry attempts
```

**Comment by bonomat** [comment]
Just to be sure: did the lnd node have funding on both sides of the channel?

**Comment by holzeis** [comment]
> Just to be sure: did the lnd node have funds on both sides of the channel?

I think so yes. As I was funding the wallet, it should not be a problem to send less money back.

**Comment by bonomat** [comment]
> I think so yes. As I was funding the wallet, it should not be a problem to send less money back.

that is true

**Comment by holzeis** [comment]
Labeling this ticket with MVP as it is blocking my money in the wallet.

**Comment by holzeis** [comment]
The reason for this bug might be that the network graph has not been updated yet, hence the node does not find a path to the the target.

**Comment by holzeis** [comment]
Just tested recently and I was able to send a payment in the following scenarios on mainnet.

1. Send money to my phoenix wallet
2. Send money to my lightning node (lnd). (Note, a direct channel with the coordinator exists as described in this bug)

On regtest I am able to pay an invoice generated by the coordinator (and other 10101 apps), but not an invoice generated by the lnd lightning node faucet. (No route found). Paying the other direction is working just fine though.

I checked the liquidity balances and everything looks ok.
